src/daq/Notification.cpp

```diff
@@ -103,7 +101,7 @@ void Notification::block(Info::MODE mode, const std::string image_id, const std:
         //
         if (!image) {
             std::ostringstream err;
-            err << "image " << image_id << " was not found after blocking " << TIMEOUT_SECONDS << " blocking at barrier";
+            err << "image " << image_id << " was not found after blocking " << _barrier_timeout*1000*1000 << " blocking at barrier";
```

Comment:
It seems something is wrong with this error message. Unless I got it wrong, the error message for the image with id being, say, `foo` and timeout equal to 1 would be

> image foo was not found after blocking 1 blocking at barrier